php-classes/Slate/Courses/SectionTermDataRequestHandler.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Slate\Courses;
+
+class SectionTermDataRequestHandler extends \RecordsRequestHandler
+{
+    public static $recordClass = SectionTermData::class;
+
+    public static $accountLevelBrowse = 'Staff';
+    public static $accountLevelWrite = 'Staff';
+    public static $accountLevelRead = 'Staff';
+    public static $accountLevelAPI = 'Staff';
+
+    public static $userResponseModes = [
+        'application/json' => 'json'
+    ];
```

Comment:
@nbey RecordsRequestHandler already sets userResponseModes, you can delete this (and should because it disables the CSV output)